aait/backend/group-2A/WebApi/Controllers/UserController.cs

```diff
@@ -16,13 +20,39 @@ public UserController(IMediator mediator)
             _mediator = mediator;
         }
 
-        [HttpPost("register")]
-        public async Task<IActionResult> CreateUser(CreateUserDTO userDto)
+        [HttpPost("Register")]
+        public async Task<IActionResult> CreateUser( CreateUserDTO userDto)
```

Comment:
To deserialize the JSON data from the request body and bind it to the CreateUserDTO object, you have to include [FromBody] to the CreateUser function.